angrmanagement/ui/widgets/qsymexec_graph.py

```diff
@@ -112,90 +88,65 @@ def show_selected(self):
     # Event handlers
     #
 
-    def mousePressEvent(self, event):
-        """
-
-        :param QMouseEvent event:
-        :return:
-        """
-        btn = event.button()
-        if btn == Qt.LeftButton:
-            pos = event.pos()
-            block = self._get_block_by_pos(pos)
-            if block is not None:
-                block.on_mouse_pressed(btn, self._to_graph_pos(pos))
-                if block.selected:
-                    self.state.am_obj = block.get_state()
-                else:
-                    self.state.am_obj = None
-                self.state.am_event(src='qsymexec_graph')
-
-                event.accept()
-                return
-
-        super(QSymExecGraph, self).mousePressEvent(event)
-
-    def mouseDoubleClickEvent(self, event):
-        """
-
-        :param QMouseEvent event:
-        :return:
-        """
-
-        btn = event.button()
-        if btn == Qt.LeftButton:
-            pos = event.pos()
-            block = self._get_block_by_pos(pos)
-            if block is not None:
-                block.on_mouse_doubleclicked(btn, self._to_graph_pos(pos))
-                event.accept()
-                return
-
-        super(QSymExecGraph, self).mouseDoubleClickEvent(event)
-
-    def _on_keypressed_event(self, key_event):
+    # def mousePressEvent(self, event):
```

Comment:
delete instead of commenting out. we have git history if we really need to see what used to be